<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UiMainWindow</class>
 <widget class="QMainWindow" name="UiMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>GomTV leacher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLineEdit" name="searchField"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="loadMore">
        <property name="text">
         <string>Load More</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionOptions"/>
   </widget>
   <widget class="QMenu" name="menuLogs">
    <property name="title">
     <string>Logs</string>
    </property>
    <addaction name="actionLogs"/>
   </widget>
   <addaction name="menuOptions"/>
   <addaction name="menuLogs"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOptions">
   <property name="text">
    <string>options</string>
   </property>
  </action>
  <action name="actionLogs">
   <property name="text">
    <string>logs</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
